spec/moonshot/build_mechanism/github_release_spec.rb

```diff
@@ -189,5 +189,27 @@ module Moonshot # rubocop:disable ModuleLength
         expect { subject.send(:check_ci_status, sha) }.to raise_error(Shell::CommandError)
       end
     end
+    
+    describe '#validate_commit' do
+      let(:subject_skip_ci) {
```

Comment:
A better format for this would be to define (let) skip_ci as false (default) and have the subject use that value.  Place the example for that directly below.  Then create a context block below that and override skip_ci to be true and place the example for skip_ci being true in that context.  Pseudo-code:
```
describe '#validate_commit' do
let(:skip_ci) { false }
subject { described_class.new(build_mechanism, skip_ci_status: skip_ci); ... }

it 'calls check_ci_status' do
  ...
end

context 'when skip_ci_status is true' do
  let(:skip_ci) { true }

  it 'does not call check_ci_status' do
    ...
  end
end
```

Reply:
Looks better and more readable for me 👍  now Done